MAiD - Case Management App/force-app/main/default/objects/Form_1634__c/fields/If_Other_service_specify__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>If_Other_service_specify__c</fullName>
+    <encryptionScheme>None</encryptionScheme>
+    <externalId>false</externalId>
+    <label>If Other service, specify</label>
+    <length>3000</length>
```

Comment:
@RanadheerRG: Was this called out by customer that it should only be 3k characters in length? Any particular reason why we should have it as 32,678/131,072? Would like to know.

Reply:
I have updated the length to 3000